### PR TITLE
[REF] base_automation: Add logger when automated action based on-time starts and ends

### PR DESCRIPTION
**Pull request by moylop260**
If there is cron worker timeout logger error
We can look for last cron running
If cron "Base Action Rule: check and execute" is the last one
So, we need to know what is the last base automated action based on-time running

This logger helps to looked for it

### CODE REVIEW

**Comment by moylop260** [comment]
@nim-odoo @mart-e 
What do you think about this `_logger.info`?

File: addons/base_automation/models/base_automation.py [hunk 1]
```diff
@@ -474,6 +474,7 @@ def _check(self, automatic=False, use_new_cursor=False):
         # retrieve all the action rules to run based on a timed condition
         eval_context = self._get_eval_context()
         for action in self.with_context(active_test=True).search([('trigger', '=', 'on_time')]):
+            _logger.info("Starting automated action based on time `%s`.", action.name)
```
Comment (AntoineVDV) [inline addons/base_automation/models/base_automation.py:477]:
```suggestion
            _logger.info("Starting time-based automated action `%s`.", action.name)
```

File: addons/base_automation/models/base_automation.py [hunk 2]
```diff
@@ -503,6 +504,7 @@ def _check(self, automatic=False, use_new_cursor=False):
                         _logger.error(traceback.format_exc())
 
             action.write({'last_run': now.strftime(DEFAULT_SERVER_DATETIME_FORMAT)})
+            _logger.info("Automated action based on time `%s` done.", action.name)
```
Comment (AntoineVDV) [inline addons/base_automation/models/base_automation.py:507]:
```suggestion
            _logger.info("Time-based automated action `%s` done.", action.name)
```

**Comment by moylop260** [comment]
Changes applied
Thanks for review it!

**Comment by mart-e** [comment]
Thanks !

@robodoo r+

**Pull request closed without merging**